spec/features/auth/provider_user_signs_in_spec.rb

```diff
@@ -12,6 +12,20 @@
     and_i_cannot_access_the_support_interface
   end
 
+  context "publish migration" do
+    before do
+      allow_any_instance_of(ActionDispatch::Request).to receive(:referer).and_return(Settings.publish_url)
+      disable_features(:send_request_data_to_bigquery)
+      enable_features(:display_migration_signin)
+    end
```

Comment:
Minor style comment — perhaps this could be captured in a single `given` step, and we drop the context block?

Reply:
done